docs/peer_review_checklist.md

```diff
@@ -56,13 +56,17 @@
 - [ ] Is the logic in the right place?
 - [ ] If the code implements functional behavior that should occur regardless of the theming, it should be in a module. (We have not been good about this).
 - [ ] If it is purely presentational, can the logic be in the template? (unless it is Mass.gov specific logic and a Mayflower template).
+- [ ] If the Drupal core was updated - does all changes from the scaffold files were applied to our version controlled files as well? (.htaccess, settings.php, services.yml, development.services.yml etc.)
 
 ## Mayflower Integration
 All openmass feature development PRs that require Mayflower changes must pass all these checks before **squashing and merging** into develop:
 - [ ] Both the Mayflower and Openmass PRs reviewed and approved
 - [ ] **Squash and merge** the corresponding Mayflower PR into develop
-- [ ] After the Mayflower develop circleCI tasks are done, in the Openmass branch, resolve merge conflicts with the develop branch and run `composer require massgov/mayflower-artifacts:dev-develop`. 
+- [ ] After the Mayflower develop circleCI tasks are done, in the Openmass branch, resolve merge conflicts with the develop branch and run `composer require massgov/mayflower-artifacts:dev-develop`.
 - [ ] Commit and push up the composer.lock changes. (mayflower-artifacts should be pointing to the latest `dev-develop` version)
 - [ ] All circleCI tests are green on the Openmass branch
 
+## JavaScript/CSS update
+- [] If JavaScript/CSS were modified - does version of the definition library incremented?
```

Comment:
```suggestion
- [] If JavaScript/CSS files were modified, does the version of the library definition need to be incremented?
```
Just thinking about this one, are you referring to the version defined here? docroot/modules/custom/mayflower/mayflower.libraries.yml

Reply:
@tom-fleming  I meant changes in all *.libraries.yml which we maintain. In custom modules and in our custom themes. And if JS/CSS were changed - the library version MUST be changed, there are no “should”. We always must bump the library definition version if CSS/JS was changed after 10.1.x otherwise it can lead to performance / aggregation issues (actually that what we have now).  I am not 100% that this is the only root of the problem, but it can one of it.